views/js/qtiCreator/plugins/panel/pciManager.js

```diff
@@ -39,72 +36,6 @@ define([
          * Initialize the plugin (called during runner's init)
          */
         init : function init(){
```

Comment:
This whole file is looking pretty empty now, no? Now that the pciManager is no longer operating as a plugin of qtiCreator, I believe this can be removed, and maybe some other code too. Please test what you can remove, and discuss with me or @ssipasseuth if you have doubts.